### PR TITLE
Add cuav  gps

### PR DESCRIPTION
**Pull request by cuhome**
add cuav  neo 3  and  neo 3 pro GPS

### CODE REVIEW

File: common/source/docs/common-cuav-neo-3-pro.rst [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+ 
+-  **GNSS Bands**
+
+   - GPS/QZSS L1 C/A,GLONASS L10F ,BeiDou B1I,Galileo E1B/C,SBAS L1 C/A: WAAS, EGNOS, MSAS, GAGAN
+
+**Number of concurrent GNSS**
+
+   - 4 Simultaneous reception of satellite information
+
+** Number of satellites(MAX)**
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-neo-3-pro.rst:40]: remove space before Number to get the format you want

File: common/source/docs/common-cuav-neo-3.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
+.. _common-cuav-neo-3-overview:
+
+==============
+NEO 3  GPS
+==============
+
+Neo 3 is a UAV positioning module developed by  CUAV. It integrates industrial-grade compass, barometer, flight controller status light, buzzer, and safety switch all in one package. It has the characteristics of high safety and strong interference resistance.It has the characteristics of high safety and strong interference resistance.It integrates ublox m9n module and can receive signals from GPS, Galileo, GLONASS and Beidou at the same time.</br>
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-neo-3.rst:7]: duplicated 3rd and 4th sentences

Comment (Hwurzburg) [reply common/source/docs/common-cuav-neo-3.rst:7]: the html tag is not needed at the end

File: common/source/docs/common-cuav-neo-3-pro.rst [hunk 3]
```diff
@@ -0,0 +1,235 @@
+.. _common-cuav-neo-3-pro-overview:
+
+==============
+NEO 3 Pro GPS
+==============
+
+NEO 3 pro is UAVCAN standard GPS developed by CUAV. It integrates industrial-grade compass, barometer, flight controller status light, buzzer, and safety switch all in one package. It has the characteristics of high safety and strong interference resistance.It integrates ublox m9n module and can receive signals from GPS, Galileo, GLONASS and Beidou at the same time.</br>
```
Comment (Hwurzburg) [inline common/source/docs/common-cuav-neo-3-pro.rst:7]: the html tag is not needed at the end

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by cuhome** [comment]
@Hwurzburg I have modified it, please check.

File: common/source/docs/common-cuav-neo-3-pro.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-cuav-neo-3-pro.rst:40]: sorry...before the word "Number"

**Comment by Hwurzburg** [comment]
can you change the last commit message? {git rebase -i HEAD~1.....then mark the commit for **r**eword, ctrl-x, fix the wording to something like ..."correct typos", ctrl-X }

File: common/source/docs/common-cuav-neo-3-pro.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-cuav-neo-3-pro.rst:40]: Sorry meant the line with the word Number

**Comment by cuhome** [comment]
@Hwurzburg I used tortoisegit ui to edit it, I don't know how to do what you described, I made a new modification, please help me to check it.

**Comment by Hwurzburg** [comment]
no problem...I fixed it....merging...thanks!

**Pull request merged**